cmd/metricswatcher/config/config.go

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
 // Copyright (c) 2019 Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information
```

Comment:
This got doubled somehow.

Reply:
@streamer45 - yeah, I think that's the custom :)
I'll fix the duplicate.
Also.. Welcome back!